cumulusci/robotframework/CumulusCI.py

```diff
@@ -114,6 +116,45 @@ def login_url(self, org=None):
             org = self.keychain.get_org(org)
         return org.start_url
 
+    def _refresh_community_info_cache(self):
+        response = self.sf.restful("connect/communities")
+        self._community_info_cache = {
+            community["name"]: community for community in response["communities"]
+        }
+
+    def get_community_info(self, community_name, key=None, force_refresh=False):
+        """This keyword uses the Salesforce API to get information about a community.
+
+        This keyword requires the exact community name as its first argumment.
+        - If no key is given, all of the information returned by the API will be
+          returned by this keyword in the form of a dictionary
+        - If a key is given, only the value for that key will be returned.
+
+        Some of the supported keys include name, siteUrl, and
+        loginUrl. For a comprehensive list see the API documentation,
```

Comment:
Do we want to include the actual doc link? https://developer.salesforce.com/docs/atlas.en-us.chatterapi.meta/chatterapi/connect_responses_community.htm

Reply:
Good idea. I'll do that.